src/components/ResultDisplay.tsx

```diff
@@ -69,3 +72,14 @@ export function ResultDisplay({ result }: ResultDisplayProps) {
 function counted(count: number, text: string) {
 	return `${count} ${text}${count === 1 ? "" : "s"}`;
 }
+
+function summed(result: EstimatedPackage[]) {
+	let total = 0;
+	result.forEach((num) => {
+		total += num.estimatedMoney;
+	});
+	const formattedTotal = total.toLocaleString("en-US", {
```

Comment:
[Praise] The `toLocaleString` is a nice touch! I was worried for usernames like `ljharb`  who have weird fractional numbers that we'd have to do something. This is that thing. 🚀